templates/account/update-customer-to-vendor.php

```diff
@@ -38,7 +38,7 @@
             <label for="seller-url" class="pull-left"><?php esc_html_e( 'Shop URL', 'dokan-lite' ); ?> <span class="required">*</span></label>
             <strong id="url-alart-mgs" class="pull-right"></strong>
             <input type="text" class="input-text form-control" name="shopurl" id="seller-url" value="<?php echo esc_attr( $shop_url ); ?>" required="required" />
-            <small><?php echo home_url() . '/' . dokan_get_option( 'custom_store_url', 'dokan_general', 'store' ); ?>/<strong id="url-alart"></strong></small>
+            <small><?php echo untrailingslashit( home_url() ) . '/' . dokan_get_option( 'custom_store_url', 'dokan_general', 'store' ); ?>/<strong id="url-alart"></strong></small>
```

Comment:
It would be good if you sanitise the url using `esc_url`. For example
```
<small>
    <?php 
    $home_url = untrailingslashit( home_url() );
    $custom_store_url = dokan_get_option( 'custom_store_url', 'dokan_general', 'store' );
    $full_url = esc_url( $home_url . '/' . $custom_store_url );
    echo $full_url; 
    ?>/
    <strong id="url-alart"></strong>
</small>